R/step_lag_difference.R

```diff
@@ -16,11 +16,12 @@
 #' @export
 #' @examples
 #' r <- epi_recipe(case_death_rate_subset) %>%
-#'   step_lag_difference(case_rate, death_rate, horizon = c(7, 14))
+#'   step_lag_difference(case_rate, death_rate, horizon = c(7, 14)) %>%
+#'   step_epi_naomit()
 #' r
 #'
 #' r %>%
-#'   recipes::prep() %>%
+#'   recipes::prep(case_death_rate_subset) %>%
 #'   recipes::bake(case_death_rate_subset)
```

Comment:
```suggestion
#'   prep(case_death_rate_subset) %>%
#'   bake(case_death_rate_subset)
```

Thanks! I'm not sure whether the prefixes are needed here (I think/hope not). May as well ditch them if possible at the same time.

Reply:
Sure, seems to work